### PR TITLE
Create assignments page

### PR DESCRIPTION
**Pull request by moustf**
- Created the assignment page for the students and the teachers.
---
Screenshot from 2022-10-25 18-29-12
Screenshot from 2022-10-25 16-06-44
Screenshot from 2022-10-26 12-59-53

### CODE REVIEW

**Review by mahmoudhammad309 — APPROVED** [review]

File: client/src/components/ClassFeedback/ClassFeedback.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { Button, Form, Input } from "antd";
+
+const ClassFeedback: React.FC = () => {
+  const onFinish = (values: any) => {
+    console.log("Success:", values);
+  };
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log("Failed:", errorInfo);
+  };
```
Comment (Raghad-Mezied) [inline client/src/components/ClassFeedback/ClassFeedback.tsx:11]: Add its logic

File: client/src/components/Class/cards/TeacherAssignmentCard.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  const [openFirst, setOpenFirst] = useState(false);
+  const [openSecond, setOpenSecond] = useState(false);
+  const [submittedAssignment, setSubmittedAssignment] = useState<Array<object>>(
+    []
+  );
+  const [notSubmittedAssignment, setNotSubmittedAssignment] = useState<
+    Array<object>
+  >([]);
+
+  const classId = 5; // ? This will change depending on the class that is clicked on.
```
Comment (Raghad-Mezied) [inline client/src/components/Class/cards/TeacherAssignmentCard.tsx:23]: Update it to dynamic id

File: client/src/components/Class/Assignments/Assignments.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+const { Search } = Input;
+
+const Assignments: React.FC = () => {
+  const [assignments, setAssignments] = useState<Array<object>>([]);
+  const [addTest, setAddTest] = useState<boolean>(false);
+  const [, setAddAssignment] = useState<boolean>(false);
+
+  const classId = 5; // ? This will be passed when the user clicks on some class.
+  const role = "teacher";
```
Comment (Raghad-Mezied) [inline client/src/components/Class/Assignments/Assignments.tsx:22]: Update the static values

File: client/src/components/Class/Assignments/Assignments.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+          <section className="assignments-box">
+            {assignments.map((assignment: any) => (
+              <StudentAssignmentCard
+                title={assignment.title}
+                createdAt={assignment.createdAt}
+                description={assignment.description}
+              />
+            ))}
+          </section>
+        )} */}
```
Comment (Raghad-Mezied) [inline client/src/components/Class/Assignments/Assignments.tsx:106]: If you don't need it, delete it.

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

File: client/src/components/ClassFeedback/ClassFeedback.tsx (hunk 1 above)
Comment (moustf) [reply client/src/components/ClassFeedback/ClassFeedback.tsx:11]: This component has been replaced with the Feedback component page in the Class.

**Pull request merged**